js/gameConfig.js

```diff
@@ -187,6 +194,17 @@ const tooltips = {
     'Library': 'Stores a collection of books, each containing vast amounts of information from basic life skills to complex magic spells.',
 };
 
+const layerData = {
+    0: new LayerData('#ffe119'),
+    1: new LayerData('#f58231'),
+    2: new LayerData('#e6194B'),
+    3: new LayerData('#911eb4'),
+    4: new LayerData('#4363d8'),
+    5: new LayerData('#47ff00'),
+}
```

Comment:
Puh, wenn eh alle Layer angegeben werden sollen und Level eh numerisch sind --> mach aus dem Object einfach nen Array

```suggestion
const layerData = [
    new LayerData('#ffe119'),
    new LayerData('#f58231'),
    new LayerData('#e6194B'),
    new LayerData('#911eb4'),
    new LayerData('#4363d8'),
    new LayerData('#47ff00'),
];
```

Und Semikolon am Ende von Statements bitte nicht vergessen.